app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -104,21 +136,15 @@ const noop = () => {};
 /**
  * Returns true if the given Acuant capture failure was caused by the user declining access to the
  * camera, or false otherwise.
- *
- * @param {import('./acuant-camera').AcuantCaptureFailureError} error
- *
- * @return {boolean}
  */
-export const isAcuantCameraAccessFailure = (error) => error instanceof Error;
+export const isAcuantCameraAccessFailure = (error: AcuantCaptureFailureError) =>
+  error instanceof Error;
```

Comment:
should we make this a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates)? which can sometimes help clean up type casting around callsites of this function

```suggestion
export const isAcuantCameraAccessFailure = (error: AcuantCaptureFailureError): error is Error  =>
  error instanceof Error;
```